_episodes/02-practice-learning.md

```diff
@@ -75,61 +75,99 @@ tasks with normal effort under normal circumstances.
-implications for how we teach novices.
-
-## Cognitive Development and Mental Models
-
-Effective learning is facilitated by the creation of a well-founded mental model. A mental model is a collection of concepts and facts,
-along with the relationships between those concepts, which a person has about a topic. For example, a long-time resident of the United
-States may have an advanced understanding of the location of US states, major cities and landmarks, weather patterns, regional
-economies and demographic patterns, as well as the relationships among these, compared with their understanding of these relationships
-for other countries. In other words, their mental model of the United States is more complex compared with their mental model of other
-countries.
-
-We can distinguish between a *novice* and a *competent
-practitioner* for a given domain based on the complexity of their mental models.
-
-*     A *novice* is someone who has not yet built a mental model of the domain.
-They therefore reason by analogy and guesswork,
-borrowing bits and pieces of their mental models of other domains
+as this tends to characterize The Carpentries audience.
+
+It is common to think of a novice as a sort of an "empty vessel" into which knowledge can be "poured." Unfortunately, this analogy has
+limited usefulness and is wrong in ways that generate dangerous misconceptions. In our next section, we will briefly explore the nature of "knowledge" through a concept
+that helps us differentiate between novices and competent practitioners in a more useful and visual way. This, in turn, will have implications
+for how we teach.
+
+## "All Models are Wrong, but Some are Useful"
+
+"Knowledge" is hard to describe. Understanding is never a mirror of reality, even for an expert; rather, it is 
+an internal representation based on our experience with a subject. 
+This internal representation is often described as a **mental model**. A mental model
+allows us to extrapolate, or make predictions beyond and between the narrow limits of experience and memory, filling in 
+gaps to the point that things "make sense." 
+
+As we learn, our mental model evolves to become more useful. A useful model makes reasonable predictions and fits well within
+the range of things
+we are likely to encounter, or at least does not break down entirely as new concepts are added. A driver of a gasoline powered car may 
+do just fine with a mental model that includes relationships between fuel, an engine, and a car battery;
+a broken alternator may not be something they could predict, but learning that another part mediates the relationship 
+between the engine and the battery adds to the model without the driver having to add any other concepts or un-learn mis-conceived relationships. 
+Alternatively, a child who attributes sentient will to a vehicle will have to work much harder to make sense 
+of a broken alternator.
```

Comment:
this is hard to parse for me at a couple of levels:
- I think a little bit of grammatical revision could make this example more readable
- I'm not sure what the takeaway is supposed to be here and what the most salient parts of this example are. 



I appreciate the goal of not using something programming related that relies on a. more sophisticated mental model than what we teach, because it can leave out some trainees.  At the same time, something a little closer can also start to get trainees to start thinking in terms of this.  Maybe we can source from the trainer community a few examples here?  I think this is a place where people use their own examples often anyway.

Reply:
this could be my relative comfort with concept maps, but i also feel like using this example here to show a very simple concept map -- not trying to teach them how to write their own yet, jsut *using* one to externalize the mental model -- could be helpful here

Reply:
I'm going to interpret this comment to indicate that the restructuring mostly works, but the example and the mechanics of connection between mental models and concept maps may need some work. On this basis, I plan to let this sit for a while while I work on other sections and think about these comments. :) Please let me know if I have that wrong!

I struggled to come up with an example that really did what I wanted it to do, which is to present a sort of mechanical causality that was 1) generally relatable, 2) simple and clear both with and without a missing element, and 3) included a misconception that would obviously interfere with understanding the missing element. Other examples I have seen Trainers use do a great job of illustrating concept mapping or mental models but don't check all these boxes. However, there are certainly other examples that may work better, including some that are more proximal, like relationships between hardware components or maybe the difference between streaming services and DVDs... things that break (or don't) when the internet fails? Will keep thinking and consider crowd-sourcing!

Reply:
As an aside, I think my experiment with task decomposition instead of concept maps achieved my objectives. Not sure if it's worth bringing it in as an alternate during this revision cycle though.

Reply:
I think the formerly Blooms-based activity in Episode 17 now resembles this task decomposition activity.